### PR TITLE
CANDLEPIN-497: Remove Ruby setup from ansible and add Gemfiles back

### PR DESCRIPTION
**Pull request by wottop**
The best way to test the docker image build is to apply this diff to the branch:
http://pastebin.test.redhat.com/1093124
You have to copy the Gemfile and Gemfile.lock to the docker/ directory before building images locally.
The real image building only happens after it is all in master.

### CODE REVIEW

File: docker/candlepin-base-cs8/setup-devel-env.sh [hunk 1]
```diff
@@ -61,23 +61,6 @@ BASHRC
 git clone https://github.com/candlepin/candlepin.git /candlepin
 cd /candlepin
 
-# Setup and install rvm, ruby and pals
```
Comment (nikosmoum) [inline docker/candlepin-base-cs8/setup-devel-env.sh:64]: We can't remove ruby from our c7 and cs8 containers. We need it to run rspec in older branches.

File: ansible/roles/candlepin/README.md [hunk 2]
```diff
@@ -13,14 +13,6 @@ Candlepin has many requirements for its developers. You will need a Fedora or Ce
 enabled to use this role. It is recommmended that that box has a non-root user to run Candlepin on (see
 variables below).
 
-Dependencies
-------------
-
-This role depends on the `rvm1.ruby` role for installing and configuring RVM. At the time of writing, Candlpin
```
Comment (nikosmoum) [inline ansible/roles/candlepin/README.md:19]: we also need to remove the reference of the role from `ansible/requirements.yml`

**Review by nikosmoum — CHANGES_REQUESTED** [review]
ACK on the removal of ruby from ansible scripts
NACK on its removal from the containers. We still need them there to run rspec on older supported branches.

**Comment by nikosmoum** [comment]
@wottop If the changes in the pastebin need are needed (copying of the Gemfiles), should they not be applied on this PR?

**Comment by wottop** [comment]
> @wottop If the changes in the pastebin need are needed (copying of the Gemfiles), should they not be applied on this PR?

That is the workaround for the fact that a local build will retrieve the master branch to build the image.

**Review by nikosmoum — APPROVED** [review]

**Pull request merged**